.deploy/nais/app-preprod.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: "nais.io/v1alpha1"
 kind: "Application"
 metadata:
-  name: familie-ba-mottak
+  name: familie-baks-mottak
```

Comment:
Ved å overskrive fss oppsett i deploy og githubs, så vil man ikke få deployet fikser til fss

Reply:
Kan en mulig løsning være å en liten periode deploye denne branchen helt til prod-gcp, så har vi mulighet til å legge ut fikser på ba-mottak må master hvis det skulle være behov for det?

Reply:
Det vil fungere